plugins/outputs/datadog/datadog.go

```diff
@@ -25,11 +25,12 @@ import (
 var sampleConfig string
 
 type Datadog struct {
-	Apikey      string          `toml:"apikey"`
-	Timeout     config.Duration `toml:"timeout"`
-	URL         string          `toml:"url"`
-	Compression string          `toml:"compression"`
-	Log         telegraf.Logger `toml:"-"`
+	Apikey           string          `toml:"apikey"`
+	Timeout          config.Duration `toml:"timeout"`
+	URL              string          `toml:"url"`
+	Compression      string          `toml:"compression"`
+	RateInterval     int64           `toml:"rate_interval"`
```

Comment:
Wouldn't it be easier for the user to configure it as a duration?
```suggestion
	RateInterval     config.Duration `toml:"rate_interval"`
```

Reply:
Good suggestion, I added that in https://github.com/influxdata/telegraf/pull/15702/commits/d4e1381bd4f5724c2d5c136467a6c9118f269e86 - but I am unsure how to make this better, given that `d.RateInterval.Seconds()` is not available to `config.Duration`. I appreciate any tips you might have 👍🏻 

```go
						// interval is expected to be in seconds
						interval = int64(d.RateInterval / 1000000000)
						dogM[1] = dogM[1] / float64(interval)
						tname = "rate"
```

Reply:
Perfect - thank you! https://github.com/influxdata/telegraf/pull/15702/commits/9bda584fa1cab8df259ed76bd3cee934b5cd88f5